op-challenger/game/monitor_test.go

```diff
@@ -97,7 +101,7 @@ func TestMonitorGames(t *testing.T) {
 					Number: big.NewInt(1),
 				}:
 				case <-ctx.Done():
-					break
+					break loop
```

Comment:
Same here, can just return.

Reply:
Also a few lines below this the `require.NoError(t, waitErr)` is redundant - could you remove it while we're in this area please?